src/index.js

```diff
@@ -190,6 +186,10 @@ const createCommandContext = (ctx, argv) => {
     const name = template.name(self.name, templateHelpers);
     const fileName = name + template.extension;
 
+    assert(validFilename(fileName), () =>
```

Comment:
@nickav Using assert which is consistent with how other checks are performed.

Reply:
Because command is no longer synonymous with print, the check is being done at the print level which makes more sense to me

Reply:
There may be a way that we can use metadata on commands to create cmd line requirements

Reply:
yeah this seems good for now though!